test/health_failures_test_case.py

```diff
@@ -0,0 +1,117 @@
+from contextlib import contextmanager
+import json
+from uuid import uuid4
+
+import boto3
+from mock import (
+    MagicMock,
+    patch,
+)
+from more_itertools import chunked
+from moto import (
+    mock_sts,
+    mock_sqs,
+    mock_dynamodb2,
+)
+import requests
+
+from app_test_case import LocalAppTestCase
+from azul import config
+from azul.health import HealthController
+from azul.json_freeze import (
+    freeze,
+    sort_frozen,
+)
+from azul.modules import load_app_module
+from retorts import ResponsesHelper
+
+
+class TestHealthFailures(LocalAppTestCase):
+    dynamo_failures_table_settings = {
+        'TableName': config.dynamo_failures_table_name,
+        'KeySchema': [
+            {
+                'AttributeName': 'MessageType',
+                'KeyType': 'HASH'
+            },
+            {
+                'AttributeName': 'SentTimeMessageId',
+                'KeyType': 'RANGE'
+            }
+        ],
+        'AttributeDefinitions': [
+            {
+                'AttributeName': 'MessageType',
+                'AttributeType': 'S'
+            },
+            {
+                'AttributeName': 'SentTimeMessageId',
+                'AttributeType': 'S'
+            }
+        ],
+        'ProvisionedThroughput': {
+            'ReadCapacityUnits': 5,
+            'WriteCapacityUnits': 5
+        }
+    }
+
+    # Moto's dynamodb backend doesn't support government regions.
+    _aws_test_region = 'ap-south-1'
+
+    @classmethod
+    def lambda_name(cls) -> str:
+        return 'service'
+
+    @contextmanager
+    def _mock_failures_table(self):
+        dynamodb = boto3.resource('dynamodb')  # , region_name=self._aws_test_region)
```

Comment:
```suggestion
        dynamodb = boto3.resource('dynamodb')
```

Reply:
```
Syntax: rate(Value Unit)
Rate expressions have two required fields. Fields are separated by white space.
Valid values: minute | minutes | hour | hours | day | days
```
https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#RateExpressions

There is an alternate `cron()` syntax however it only allows granularity to the minute as well.

I believe my latest approach with `math.ceil` gives good results:
```

0 sec -> 0 min
1 sec -> 1 min
...
53 sec -> 1 min
54 sec -> 1 min
55 sec -> 2 min
56 sec -> 2 min
...
108 sec -> 2 min
109 sec -> 2 min
110 sec -> 3 min
111 sec -> 3 min
...
162 sec -> 3 min
163 sec -> 3 min
164 sec -> 4 min
165 sec -> 4 min
...
217 sec -> 4 min
218 sec -> 4 min
219 sec -> 5 min
220 sec -> 5 min
...
271 sec -> 5 min
272 sec -> 5 min
273 sec -> 6 min
274 sec -> 6 min
...
326 sec -> 6 min
327 sec -> 6 min
328 sec -> 7 min
329 sec -> 7 min
...
380 sec -> 7 min
381 sec -> 7 min
382 sec -> 8 min
383 sec -> 8 min
384 sec -> 8 min
```